HEN_HOUSE/omega/progs/dosxyz_show/Makefile

```diff
@@ -43,13 +43,13 @@ X11_include = -I/usr/X11R6/include
 # On our system, Motif include files are in /usr/X11R6/include/Xm =>
 # we don't need a separate -I for Motif. If this is different on your
 # system, adjust accordingly.
-Motif_include =
+Motif_include = -I/opt/local/include
 
 X11_libdir = -L/usr/X11R6/lib
 # On our system, the Motif library is /usr/X11R6/lib =>
 # we don't need a separate -L for Motif. If this is different on your
 # system, adjust accordingly.
-Motif_libdir =
+Motif_libdir = -L/opt/local/lib
```

Comment:
Ok, just modify the files and then use `git commit --amend` so it replaces the existing commit, and then `git push --force fix-omega_makefiles`. Or else just push the new commit as usual, and I'll take care of it!